src/app/pages/Scan/Scan.module.css

```diff
@@ -10,14 +10,126 @@
   padding: 4rem 2rem;
 }
 
-.skip {
-  background-color: var(--color-primary);
-  color: var(--color-secondary);
-  border-radius: 8px;
-  width: 12rem;
-  height: 2.5rem;
+h1 {
```

Comment:
Be careful using element selectors. This will affect all `h1` in your project.

Reply:
done